DEVELOPMENT.md

```diff
@@ -0,0 +1,70 @@
+# Development Uploadcare Widget
+
+Clone the repository.
+
+### Enviroment
+
+You need a working Ruby 2.0.0 and above environment with [Bundler](http://bundler.io).
+
+#### Vagrant
+
+If you don't want to raise the environment on your machine, you can use [Vagrant](https://www.vagrantup.com/).
+Just [install Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/) or [other provider](https://www.vagrantup.com/docs/getting-started/providers.html).
+
+After open command line, go to this folder and run:
+
+    vagrant up
+    vagrant ssh
+
+#### Local environment (without Vagrant)
+
+Inside folder run
+
+    bundle install
+    cd ./test/dummy
+    bundle install
+
+### Build
+
+* `bundle exec rake js:latest:build` to build assets
+  to the **pkg/latest** directory (with the “latest” suffix).
+* `bundle exec rake js:release:build` to build assets
+  to the **pkg/version** folder (with the current version suffix).
+  The version is specified in `lib/uploadcare-widget/version.rb`.
+
+### Development
+
+Go to `test/dummy/`. There is a simple Rails app. Run it:
+
+    bundle exec rails server
+    
+Open http://127.0.0.1:3000/. Follow any link. 
+There's going to be a widget or three. Edit code and reload page :-)
+
+
+### Testing
+
+[Jasminerice](https://github.com/bradphelan/jasminerice) 
+installed under the `test/dummy/` Rails app.
+
+To run tests in your browser go to http://127.0.0.1:3000/jasmine.
+
+For more information see 
+[jasminerice docs](https://github.com/bradphelan/jasminerice).
+
+#### guard-jasmine
+
+To run tests in a terminal you must first 
+[install phantomjs](https://github.com/guard/guard-jasmine#phantomjs).
+
+Then you have two options:
```

Comment:
`Then you have two options: your son dies or you send us the money, biatch`

Then, there are two options: